ops/docker-compose.yml

```diff
@@ -197,6 +197,25 @@ services:
       - ${REPLICA_HTTP_PORT:-8549}:8545
       - ${REPLICA_WS_PORT:-8550}:8546
 
+  replica-healthcheck:
+    depends_on:
+      - l2geth
+      - replica
+    deploy:
+      replicas: 1
```

Comment:
I have a preference to have `replicas: 0` by default, we don't need this to run during local development, the setup is already becoming beefy

Reply:
Ok, done. Not included by default but will run in integration tests.